packages/commcare/package.json

```diff
@@ -25,8 +25,8 @@
     "configuration-schema.json"
   ],
   "dependencies": {
-    "@openfn/language-common": "workspace:1.11.1",
-    "@openfn/language-http": "workspace:^5.0.4",
+    "@openfn/language-common": "workspace:1.12.0",
+    "@openfn/language-http": "workspace:^6.0.0",
```

Comment:
This could be trouble - commcare is likely to break!

Reply:
Commcare is actually a nightmare - it uses common.http, http.post, AND superagent!!

Luckily it only uses a very simple version of http.post. So it won't break.

Reply:
There is an issue for this: https://github.com/OpenFn/adaptors/issues/414